qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VisitSteps.java

```diff
@@ -50,7 +50,7 @@ public void searchForPatientRecord() {
     	findPatientPage.waitForPageToLoad();
     }
     
-    @And("user clicks on the first patient record")
+    @And("user clicks on the patient returned in the patient table")
     public void clickTheFirstPatientRecord() {
```

Comment:
Is the test method remaining the same?

Reply:
Yes please , this remains the same.

Reply:
`clickTheFirstPatientRecord()` suggests that there is a possibility of returning more than one patient from the pool. If the search is done by using patient identifier then one patient should be returned from the search indexing criteria.

Reply:
Feel free Cross check my last commit to see if it addresses this comment.